variables.mk

```diff
@@ -88,6 +88,7 @@ KIND_WAIT := 30s
 ### ci vars
 # timeout for short test suite, must be parsable as Go time.Duration (60m, 2h)
 SHORT_CI_TIMEOUT := 60m
+SANITY_TEST := volumes should store data  # focus expects to get a regexp as input not a simple string
```

Comment:
Will that work without quotes?

Reply:
There is a problem with the following:
* if we pass variable from outside, it can't be with quotes as they processed inaccurately, so I'm adding quotes in Makefile during command construction
* if we have a variable in variables.mk with quotes, it'll be passed to Makefile with them, so we'll have doubled quotes
It doesn't work without quotes, I'm working on workaround currently.